node/src/BaseClient.ts

```diff
@@ -1224,6 +1225,33 @@ export class BaseClient {
         );
     }
 
+    /** Moves `member` from the set at `source` to the set at `destination`, removing it from the source set.
+     * Creates a new destination set if needed. The operation is atomic.
+     * See https://valkey.io/commands/smove for more details.
+     *
+     * Note: When in cluster mode, `source` and `destination` must map to the same hash slot.
```

Comment:
We have another style for cross slot notice
https://github.com/aws/glide-for-redis/blob/64d2273f093a6f27d0674f741f6ea7303b02563b/node/src/BaseClient.ts#L2224

Reply:
I was just about to ask you! thanks